src/System.IO.FileSystem.Watcher/src/packages.config

```diff
@@ -17,6 +17,7 @@
   <package id="System.Runtime.InteropServices" version="4.0.20-beta-22412" targetFramework="portable-net45+win+wpa81" />
   <package id="System.Text.Encoding" version="4.0.10-beta-22412" targetFramework="portable-net45+win+wpa81" />
   <package id="System.Threading" version="4.0.0-beta-22412" targetFramework="portable-net45+win+wpa81" />
+  <package id="System.Threading.Thread" version="4.0.0-beta-22412" targetFramework="portable-net45+win+wpa81" />
```

Comment:
No.  We are going to transfer ownership of the thread to the CFRunLoop so it can pump events, so we want a new thread.